src/js/control-bar/progress-control.js

```diff
@@ -1,3 +1,7 @@
+import Component from '../component';
+import Slider, { SliderHandle } from '../slider';
```

Comment:
I would like to actually split up the slider file to have a slider and a sliderhandle file. Same with this progress-control file and most of these controls files that have more than one component in them.